Utils/IdentifierHandlers.cs

```diff
@@ -8,13 +8,15 @@ private void SetAbilityIdentifiers()
       {
         foreach (AbilityIdentifier id in item.Value)
         {
+          var newCard = NewCard.cards[item.Key];
           if (id.id != 0)
           {
-            NewCard.cards[item.Key].abilities.Add(id.id);
+            // if the card already has the ability then no point and adding it
+            if (!newCard.abilities.Contains(id.id)) newCard.abilities.Add(id.id);
```

Comment:
Some abilities can stack. If you try to add a duplicate which can't stack, then it will be removed, but certain abilitiies stack

Reply:
A fix for toRemove does need to be added to make this work as intended but I have the code for that and will add it after I merge